civet.dev/cheatsheet.md

```diff
@@ -315,3 +315,39 @@ floor # / 2
 ```
 
 </div>
+
+## ASCII Symbols
+
+Here is a list of ASCII symbols and their various contextual meanings in Civet.
+(There are also more [Unicode symbols](/reference#unicode-operators).)
+
+| Symbol | Meanings |
+|--------|----------|
+| space | Function call |
+| `?` | Non-null test `x?`; non-null access `x?.y`, `x?y`, `x?(...)`, `x?[...]`; null coalescing `x ?? y` (`x ? y` with `coffeeBinaryExistential`); TypeScript optionals `(x?: number) => x`; if/then/else ternary `x ? y : z` |
+| `!` | Negation `!x`; object flag `{!x}`; TypeScript non-null assertion `x!`; non-null assertion access `x!.y`, `x!y` |
+| `@` | this shorthand `@`; this properties `@x`; method bind `x@y`, `x@.y`, `@@x`; decorators `@@d` |
+| `#` | private properties `x.#y`, `#y`; length shorthand `x#`, `#`; block comments `### ... ###`; one-line comments `# ...` with `coffeeComment`; CoffeeScript string interpolations `"x#{y}"` with `coffeeInterpolation` |
+| `$` | JavaScript identifier character; template interpolations `` `x${y}` `` |
+| `_` | JavaScript identifier character |
+| `%` | Modulo operators `x % y`, `x %% y`; modulo indexing `x[i %]`; integer division `%/` |
+| `^` | Binary XOR `x ^ y`; logical XOR `x ^^ y`; logical XNOR `x !^ y`, `x !^^ y`; pin patterns `(^x) =>` |
+| `&` | Binary AND `x & y`; logical AND `x && y`; TypeScript AND `number & string`; placeholder for function shorthand `&[x]+1` |
+| `\|` | Binary OR `x \| y`; logical OR `x \|\| y`; TypeScript OR `number \| string`; pipeline `\|>`, `\|\|>` |
+| `+` | Addition `x + y`; positive `+x`; increment `x++`, `++x`; object flag `{+x}`; concatenation `x ++ y` |
+| `-` | Subtraction `x - y`; negation `-x`; decrement `x--`, `--x`; object flag `{-x}`; function arrows `->`; property name symbol `x-y: z` |
+| `*` | Multiplication `x * y`; exponentiation `x ** y`; generators `function*`, `*method()`, `for*`; comments `/* x */` |
```

Comment:
We might want to explicitly mention `yield*` as a special thing relating to generators